recipes/install.rb

```diff
@@ -115,6 +119,13 @@
   not_if { node['install']['external_users'].casecmp("true") == 0 }
 end
 
+group "docker" do
+  action :modify
+  members [node['hops']['yarnapp']['user']]
+  append true
+  not_if { node['install']['external_users'].casecmp("true") == 0 }
+end
```

Comment:
I'm a bit confused here. I don't see this requirement in the documentation, and here we are basically granting yarnapp sudo privileges. This is going to be a problem in security reviews. Also, if I remember correctly, you told me that users can still run non docker applications. If that's so, this is the gateway for taking over a node.

Reply:
I don't remember why I do that. I need to check, it is probably a mistake.